CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-gazebo3 VERSION 3.15.0)
+project(ignition-gazebo3 VERSION 3.16.0)
```

Comment:
3.15.1? All the PRs seem like bug fixes and no API changes.

Reply:
Done.